lib/util/http/auth.ts

```diff
@@ -1,5 +1,5 @@
 import is from '@sindresorhus/is';
-import type { NormalizedOptions } from 'got';
+import type { Options } from 'got';
```

Comment:
Why this?

Reply:
We're about to delete `username` and `password` fields, which are mandatory for `NormalizedOptions`.
Another option is to set those two to empty strings instead, but I've tried not to change any test snapshots.